content/docs/engine-dev/assets/filetypes/texture-support.md

```diff
@@ -0,0 +1,113 @@
+* Scene files should emit job dependencies on referenced images.
+
+There are other potential concerns that are lower priority, and can be worked around:
+
+* UI references to images via SimpleAssetReference
+* Material Types reference images
+
+## Asset Relocation Challenges
+### Materials authored in the material editor have path based references to images
+
+Materials generally come from two places right now: Material product assets (azmaterials) generated from scene files, and materials authored in the material editor.
+
+This concern is on the authored materials. Asset relocation issues around textures for materials generated from scene files is less of a concern because that same workflow challenge exists in other engines with sidecar based relocation.
+
+Material source assets reference image source assets via relative path.  These are emitted as job dependencies, so we have more hooks available for us to track and manage this.  However, right now, it's not handled. For example, renaming `bcurtain_metallic.png` only the reprocesses the image and not the material.
+
+This means that right now, the product asset remains the same. It continues to have an asset ID based reference to the image asset, with the old incorrect asset hint.
+
+Because there's a job dependency here, the job would re-process in this case, and not just silently point at the old link (see the other concerns with relocation in this document)
+
+Forcing the material to reprocess causes an error to be emitted.
+
+#### So why is this concern?
+
+It means that, when someone renames an image file referenced via material source assets, all of those jobs will fail until each of those material files are individually updated, even with a stable asset ID. So in this case, when renaming `curtain_metallic.png`, all 3 materials that reference `curtain_metallic.png` by relative path will need to be individually updated. I now have to modify 3 additional source assets:
+
+Instead, if the image was referenced via UUID or asset ID then this rename event would mean that the materials could remain unchanged and would limit the scope of work to just that individual image being renamed, and not the incoming dependencies.
+#### What can we do to address this?
+
+* Update materials to reference images via source asset UUID, or product asset asset ID.
+* Build a migration tool that users can run that will update all existing materials on their project to the new material format, so they don't have to manually re-open and re-save all materials.
+* In the guidance for enabling meta sidecar files for images, explain this situation to users, and ask them to run this tool before enabling meta files for image types.
+
+This will mean that material references to images will be stable across image renaming, because materials will use stable IDs.
+### Scene file source assets do not emit job dependencies on images reference by path

```

Comment:
```suggestion
This will mean that material references to images will be stable across image renaming, because materials will use stable IDs.

### Scene file source assets do not emit job dependencies on images reference by path

```

Nit: Another nit picky blank new line between a body line and a title block.